README.md

```diff
@@ -954,3 +954,29 @@ Nexus instead. The modified files will be accessible at the
 again in a future request, it will use it directly from Nexus rather than downloading it and
 uploading it again. This guarantees that any dependency used for a Cachito request can be used again
 in a future Cachito request.
+
+## Using Cachito Without Package Managers
+
+Cachito can be used without specifying a package manager in a request. In that case, only the source code
+present in the specified commit in a repository will be downloaded and cached.
+
+Even if there are package manager definitions in the source code (such as a `package.json` or a `requests.txt`
+file), they'll be ignored using this approach. Besides not being cached, the dependencies will also be absent
+from the content manifest.
+
+This approach can be useful in case there's need to cache and use only the actual source code for that commit.
+The tarball served by Cachito will contain all the files present in the specified commit. Here's an example of
+a request without package managers:
+
+```bash
+curl "localhost:8080/api/v1/requests" \
+    -X POST \
+    -H 'content-type: application/json' \
+    -d '{
+          "repo": "https://github.com/cachito-test/cachito-pip-with-deps/",
+          "ref": "1ca07be3001450dbc4f0224e0f763c60353d0f01"
+        }'
```

Comment:
Cachito will "helpfully" default to `"pkg_managers": ["gomod"]` if not specified. The only way to get empty package managers is explicitly `"pkg_managers": []`

Reply:
Now, isn't that a surprise :smile: 
Fixed it.